testing/functional_tests/t_utils.py

```diff
@@ -1,92 +1,27 @@
-import sys
 import os
+import pytest
 import shutil
-from ivadomed.utils import init_ivadomed, __ivadomed_dir__
-from ivadomed.scripts import download_data as ivadomed_download_data
+from ivadomed.utils import init_ivadomed
+from testing.common_testing_util import remove_tmp_dir, path_repo_root, path_temp, path_data_functional_source, \
+    path_data_functional_tmp, download_dataset
 
-__test_dir__ = os.path.join(__ivadomed_dir__, 'testing/functional_tests')
-__fixtures_dir__ = os.path.join(__test_dir__, 'fixtures')
-__data_testing_dir__ = "data_functional_testing"
-__tmp_dir__ = "tmp"
+__test_dir__ = os.path.join(path_repo_root, 'testing/functional_tests')
+__data_testing_dir__ = path_data_functional_source
+__tmp_dir__ = path_temp
 
 init_ivadomed()
 
 
-class bcolors(object):
-    """Class for different colours."""
-
-    normal = '\033[0m'
-    red = '\033[91m'
-    green = '\033[92m'
-    yellow = '\033[93m'
-    blue = '\033[94m'
-    magenta = '\033[95m'
-    cyan = '\033[96m'
-    bold = '\033[1m'
-    underline = '\033[4m'
-
-
-def printv(string, verbose=1, type='normal'):
-    """Print color-coded messages, depending on verbose status.
-
-    Only use in command-line programs (e.g. sct_propseg).
-    """
-    colors = {
-        'normal': bcolors.normal,
-        'info': bcolors.green,
-        'warning': bcolors.yellow,
-        'error': bcolors.red,
-        'code': bcolors.blue,
-        'bold': bcolors.bold,
-        'process': bcolors.magenta
-    }
-
-    if verbose:
-        # The try/except is there in case stdout does not have isatty field (it did happen to me)
-        try:
-            # Print color only if the output is the terminal
-            if sys.stdout.isatty():
-                color = colors.get(type, bcolors.normal)
-                print(color + string + bcolors.normal)
-            else:
-                print(string)
-        except Exception:
-            print(string)
-
-
-def download_dataset(dataset='data_testing', verbose=True):
-    """Download testing data from internet.
-
-    Args:
-        verbose (bool): whether or not to print
+@pytest.fixture(scope='session')
+def get_functional_test_files():
```

Comment:
This function is probably the parts that require most manual validation. 

Given our current GitHub action has manually prepared the download file, it is not able to access whether this function is failing or not. Please test this in new env without existing test files as a best precaution.